workflows/sarek.nf

```diff
@@ -854,11 +867,12 @@ workflow SAREK {
             fasta_fai,
             intervals,
             intervals_bed_gz_tbi,
-            intervals_bed_combined, // [] if no_intervals, else interval_bed_combined.bed
+            intervals_bed_combined,
```

Comment:
found out whats up with this line: the comment just got removed: can you add it backin: `// [] if no_intervals, else interval_bed_combined.bed`